Tests/TuistMigrationIntegrationTests/Utilities/TargetsExtractorTests.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import TSCBasic
+import TuistSupport
+import XCTest
+
+@testable import TuistMigration
+@testable import TuistSupportTesting
+
+final class TargetsExtractorIntegrationTests: TuistTestCase {
```

Comment:
The name of the file doesn't match the name of the test :)

Reply:
Nice catch!